critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -345,11 +352,15 @@ private void doSync() {
             }
         } finally {
             long dt = System.currentTimeMillis() - start;
+            long overheadTime = dt - byteTransferTime;
             StringBuilder sb = new StringBuilder();
             sb.append("FileSyncJob.END ").append(artifactLabel);
             sb.append(" success=").append(success);
-            sb.append(" duration=").append(dt);
             sb.append(" attempts=").append(syncArtifactAttempts);
+            sb.append(" downloadAuth=").append(downloadAuth);
+            sb.append(" downloadURL=").append(downloadURL);
```

Comment:
Critwall always uses the same auth during negotiation, so specifying `downloadAuth` specifically here is ugly for no pourpose. Just `auth`.

The `logURL` is a hack to strip off potential pre-auth tokens and log the target minoc service, so it's not consistent enough to add what I'd call "permanent logs". For success=true, it will be noise in the logs we don't care about and it won't be consistent enough over time to be searchable. For success=false, it will just be the last of several download URLs tried.

Please remove `downloaddURL` from here (just leave it in FileSyncJob.ERROR messages).

Reply:
Replaced "downloadAuth" with "auth". Removed "downloadURL" from FileSyncJob.END.